charts_flutter/test/user_managed_state_test.dart

```diff
@@ -89,7 +89,10 @@ class TestChartState extends State<TestChart> {
       defaultInteractions: false,
     );
 
-    return new GestureDetector(child: chart, onTap: handleOnTap);
+    return new Directionality(
+      textDirection: TextDirection.ltr,
+      child: new GestureDetector(child: chart, onTap: handleOnTap),
+    );
```

Comment:
You can safely omit `new`

```suggestion
    return Directionality(
      textDirection: TextDirection.ltr,
      child: GestureDetector(child: chart, onTap: handleOnTap),
    );
```